apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -103,17 +113,27 @@ function WpcomBlockEditorNavSidebar() {
 	// (using the filter) to take the user to the customer homepage or themes gallery instance.
 	// `closeLabel` can be overridden in the same way to correctly label where the user will
 	// be taken to after closing the editor.
-	const defaultCloseUrl = addQueryArgs( 'edit.php', { post_type: postType.slug } );
+
+	let defaultCloseUrl;
+	let defaultCloseLabel;
+
+	if ( launchpadScreenOption === 'full' ) {
+		defaultCloseUrl = `http://wordpress.com/setup/${ siteIntent }/launchpad?siteSlug=${ siteSlug }`;
```

Comment:
hardcoding the URL like this will give us similar issues with respecting the environment we enter from (horizon vs. production etc.) I wonder if there is another way here? I guess not if we aren't in the calypso Iframe? (and that case should be handled by previous work)

Reply:
I'm still unable to figure out a simple way to retrieve the correct environment. Since we're not in calypso, I dont think theres an existing way to do this.

Reply:
makes sense, not a blocker.